boutdata/tests/test_collect.py

```diff
@@ -31,10 +36,10 @@
     {"xguards": False, "yguards": False},
 ]
 
+
 squash_params_list = [
     (False, {}),
     (True, {}),
-    (True, {"parallel": 2}),
```

Comment:
If we change this, are we testing parallelised squashing at all? The parallel version was quite tricky to implement at all, so I'd be very hesitant to remove test coverage. On the other hand, to save time it'd be fine to reduce the number of test cases for it. Maybe just test the parallel version for a single-null and double-null (as the most common/important examples)?